<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDlg</class>
 <widget class="QDialog" name="SettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	margin:0px;
}

QDialog#SettingsDlg
{
	background-color:skyblue;
}
QStackedWidget#stackedWidget
{
	background-color:qlineargradient(spread:reflect, x1:0.488, y1:0, x2:0.494, y2:1, stop:0 rgba(151, 211, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

#settings,#model1,#model2,#model3
{
	width:50px;
	height:30px;
	border:none;
	background-color:rgb(6,163,220);
	border-radius:10px;
	color:white;
}
#settings:hover
{
	background-color:darkblue;
}
#model1:hover
{
	background-color:darkblue;
}
#model2:hover
{
	background-color:darkblue;
}
#model3:hover
{
	background-color:darkblue;
}
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>91</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="btns">
    <item>
     <widget class="QPushButton" name="settings">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="model1">
      <property name="text">
       <string>人物模型1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="model2">
      <property name="text">
       <string>人物模型2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="model3">
      <property name="text">
       <string>人物模型3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="model2Page3">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>201</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource.qrc">:/Resource/desktopRole/summerGril/0.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_model2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>模型2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="model3Page4">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>懒得做，就不放了</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingsPage1">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>程序设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>自定义桌面壁纸</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="model1Page2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>161</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource.qrc">:/Resource/desktopRole/littleBoy/0.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_model1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>模型1</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
